docs/tutorials/dronecan_px4_flight_controller.rst

```diff
@@ -444,11 +430,119 @@ of the help for the actuator_test command, as well as this example command::
+        :align: center
+        :height: 500
+
+8. Set the following configurations if not already configured properly
+
+    a. ``CAN_D1_PROTOCOL``: Set to 1 to use DroneCAN on the CAN1 port
+    b. ``CAN_D1_UC_ESC_BM``: This is a bitmask that defines the ESC indices that will be transmitted with each raw command. In this example, we will only select ``ESC 1`` for our module with ESC Index 0
+      
+       .. image:: ../_static/tutorial_images/dronecan_px4_tutorial/mission_planner_esc_bitmask.png
+        :align: center
+
+    c. ``CAN_D1_UC_NODE``: Defines the DroneCAN node ID used by the flight controller. Ensure that this value is unique from all modules connected over DroneCAN
+
+9. Make sure you have written all parameters, and reboot your flight controller
+
+Testing DroneCAN with your Module
+---------------------------------------
+
+.. warning:: Before attempting any tests that may cause the module to spin, ensure any propellers are removed from the module and that the module is safely secured.
+
+With both the module and flight controller configured, you can now ensure that the two integrate as expected.
+
+Motor Test
+^^^^^^^^^^^^^^
+.. note:: 
+    
+    If your module is using :ref:`Advanced Arming <manual_advanced_arming>` with DroneCAN, then when you connect it to a bus with a properly configured flight controller 
+    you may hear your module play its :ref:`arming song <arming_song>`. When exactly the module arms will depend on how its :ref:`arming <manual_advanced_arming>` configurations are set up. By default,
+    Vertiq modules typically will arm on 0% commands, and by default ArduPilot will send 0% commands to its connected ESCs over DroneCAN when it is disarmed. So it is likely when
+    using the default arming configurations the module may arm as soon as the flight controller is properly configured. If your module is set to 
+    :ref:`bypass arming on DroneCAN <dronecan_arming_and_bypass>`, then it will never play its arming song and does not need to arm to spin. 
+
+1. Connect your flight controller to Mission Planner
+2. Expand the Optional Hardware options under the setup tab, and select Motor Test. You will see the following
+
+    .. figure:: ../_static/tutorial_images/pwm_flight_controller/mp_motor_test.png
+        :align: center
+
+        Motor Test Screen in Mission Planner
+
+3. Connect your module's CAN connection to the flight controller, and power it on. For details on your module's CAN connections, refer to its module family page. As 
+   noted above, depending on your module's configuration, you may hear your module arm immediately after power up
+4. If your :ref:`safety switch is enabled <safety_switch_config>`, arm it
+5. Using the default throttle and duration, select Test motor A. Your module should start spinning
+6. Try some other throttle levels to see the module running at different speeds
+
+Configuring your Module with DroneCAN Via ArduPilot

```

Comment:
This is very cool. I noticed PX4 lists something similar on their documentation: https://docs.px4.io/main/en/dronecan/#qgc-cannode-parameter-configuration

Does that work for us? If so, any chance we could have a section on that in the PX4 part?

Reply:
ohhh hadn't noticed that before. I'll try it out